packages/terra-action-header/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -1,40 +1,67 @@
 @import '~terra-mixins/lib/Mixins';
 
 :global {
+  --terra-action-header-align-items: center;
   --terra-action-header-background-color: #f4f4f4;
   --terra-action-header-border-bottom: 2px solid #9b9fa1;
   --terra-action-header-border-left: 0 none;
   --terra-action-header-border-right: 0 none;
   --terra-action-header-border-top: 0 none;
   --terra-action-header-box-shadow: none;
   --terra-action-header-min-height: 2.83333rem;
-  --terra-action-header-padding: 0.25rem;
+  --terra-action-header-padding: 0;
+  --terra-action-header-flex-fill-align-self: center;
+  --terra-action-header-flex-collapse-align-self: center;
+  --terra-action-header-flex-end-align-self: stretch;
+  --terra-action-header-flex-collapse-margin: 0.5rem;
 
   --terra-action-header-color: #1c1f21;
   --terra-action-header-font-size: 1.5rem;
-  --terra-action-header-title-padding: 0;
-  --terra-action-header-title-padding-top: 0;
+  --terra-action-header-font-weight: normal;
+  --terra-action-header-title-line-height: 1.25;
+  --terra-action-header-title-padding: 0.25rem 0.25rem 0.25rem 0.25rem;
+  --terra-action-header-title-padding-top: 0.25rem;
+  --terra-action-header-buttons-plus-title-padding-left: 0;
 
-  --terra-action-header-margin-right: 0.5rem;
-  --terra-action-header-margin-left: 0.5rem;
+  --terra-action-header-left-buttons-margin-left: 0;
+  --terra-action-header-left-buttons-margin-right: 0;
+  --terra-action-header-right-buttons-margin-left: 0;  
+  --terra-action-header-previous-next-button-group-margin-left: 0;
+  --terra-action-header-previous-next-button-group-margin-right: 0.5rem;
+  --terra-action-header-buttons-plus-previous-next-buttons-margin-left: 0.5rem;
 
-  --terra-action-header-icon-top: 0;
```

Comment:
Does removing this cause theme issue with old versions?

Reply:
not removed - just moved in location - now at line 41

Reply:
good catch - reverting to previous variable name since nothing else uniquely changed (wasn't removed - had just updated to new variable)